src/curve/stable/StablePair.sol

```diff
@@ -11,7 +11,7 @@ import { GenericFactory } from "src/GenericFactory.sol";
 import { IReservoirCallee } from "src/interfaces/IReservoirCallee.sol";
 import { StableMath } from "src/libraries/StableMath.sol";
 import { StableOracleMath } from "src/libraries/StableOracleMath.sol";
-import { ReservoirPair } from "src/ReservoirPair.sol";
+import { ReservoirPair, Observation } from "src/ReservoirPair.sol";
```

Comment:
Interesting, is it necessary to re-declare the Observation import all the time?

Reply:
yes seems to be the case if we want to use the struct